Tesla.css

```diff
@@ -0,0 +1,233 @@
+body {
+    max-width: 1920px;
+    margin: 0 auto;
+    font-family: Montserrat, sans-serif;
+    font-weight: 600;
+    font-style: normal;
+}
+
+.header {
+    position: relative;
+    font-weight: 600;
+}
+
+.logo {
+    width: 166px;
+    height: 51px;
+}
+
+.link {
+    text-decoration: none;
+    color: #000;
+}
+
+.nav {
+    width: 93%;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin: 0 auto;
+    margin-top: 24px;
+}
+
+.nav__link {
+    font-size: 15px;
+    white-space: nowrap;
+    margin: 0 10px;
+}
+
+.nav__model {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 37%;
+    flex-wrap: wrap;
+    margin: 0 auto;
+}
+
+.nav__input {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 14%;
+}
+.nav__link:hover {
+    color: #E51937;
+}
+
+.nav__link-s::after {
+    content:'';
+    display: block;
+    width: 100%;
+    height: 2px;
+    background: #E51937;
+    margin-top: 6px;
+}
+
+ /* .nav.nav__link:hover::after {
+    content:'';
+    display: block;
+    width: 60px;
+    height: 2px;
+    background: #E51937;
+    margin-top: 6px;
+} */
+
+.background {
+    background: url(img/background-header.png);
+    background-repeat: no-repeat;
+    max-width: 1514px; 
+    background-size: contain;
+    background-position: center center;
+    margin: 0 auto;
+}
+
+.title {  
+    width: 371px;
+    margin: 0 auto;
+    margin-bottom: 26px;
+}
+
+.title__h1 {
+    margin: 0 auto;
+    margin-top: 74px;
+    font-weight: 600;
+    font-size: 90px;
+    margin-bottom: 9px;
+}
+
+.title__h2 {
+    margin: 0 auto;
+    font-size: 15px;
+    font-weight: 500;
+    text-align: center;
+}
+
+.image__car {
+    display: block;
+    size: contain;
+    width: 67%;
+    margin-left: auto;
+    margin-right: auto;
+    margin-bottom: 26px;
+}
+
+.box {
+    margin: 0 auto;
+    margin-bottom: 37px;
+    max-width: 800px;
+    width: 56%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.box__item {
+    display: flex;
+    align-items: center;
+}
+
+.box__img-1 {
+    width: 46px;
+    height: 34.13px;
+    margin-right: 14px;
+    padding: 0;
+}
+
+.box__img-2 {
+    width: 30px;
+    height: 50px;
+    margin-right: 11px;
+}
+
+.test__img-3 {
+    width: 22px;
+    height: 22px;
+}
+.box__item-1 {
+    margin-right: 83px;
+}
+.box__item-2 {
+    margin-right: 105px;
+}
+
+.box__text{
+    display: inline-block;
+    font-family: Montserrat;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)